Build/Scripts/EnlistmentHelperFunctions.psm1

```diff
@@ -51,69 +59,129 @@ function Get-ConfigValue() {
 function Set-ConfigValue() {
     param(
         [Parameter(Mandatory=$false)]
-        [ValidateSet("BuildConfig","AL-GO")]
+        [ValidateSet("BuildConfig","AL-GO", "Packages")]
         [string]$ConfigType = "AL-GO",
         [Parameter(Mandatory=$true)]
         [string]$Key,
         [Parameter(Mandatory=$true)]
-        [string]$Value
+        $Value
     )
 
-    if ($ConfigType -eq "BuildConfig") {
-        $ConfigPath = Join-Path (Get-BaseFolder) "Build/BuildConfig.json" -Resolve
-    } else {
-        $ConfigPath = Join-Path (Get-BaseFolder) ".github/AL-Go-Settings.json" -Resolve
+    switch ($ConfigType) {
+        "BuildConfig" {
+            $ConfigPath = Join-Path (Get-BaseFolder) "Build/BuildConfig.json" -Resolve
+        }
+        "AL-GO" {
+            $ConfigPath = Join-Path (Get-BaseFolder) ".github/AL-Go-Settings.json" -Resolve
+        }
+        "Packages" {
+            $ConfigPath = Join-Path (Get-BaseFolder) "Build/Packages.json" -Resolve
+        }
     }
+
     $BuildConfig = Get-Content -Path $ConfigPath -Raw | ConvertFrom-Json
     $BuildConfig.$Key = $Value
     $BuildConfig | ConvertTo-Json -Depth 100 | Set-Content -Path $ConfigPath
 }
 
+function Get-PackageLatestVersion() {
```

Comment:
The function name doesn't really reflect what the function does. It doesn't get the latest version but the latest version for a major.minor

Reply:
Documented it.